packages/renderer/src/lib/image/RunImage.svelte

```
@@ -26,95 +27,71 @@ import Route from '/@/Route.svelte';
   const containerAlreadyExists = $containersInfos.find(
     container =>
       container.engineId === imageInspectInfo.engineId &&
-      container.Names.some(iteratingContainerName => iteratingContainerName === `/${containerName}`),
+      container.Names.some(iteratingContainerName => iteratingContainerName === `/${options.basic.containerName}`),
   );
   if (containerAlreadyExists) {
-    return `The name ${containerName} already exists. Please choose another name or leave blank to generate a name.`;
+    return `The name ${options.basic.containerName} already exists. Please choose another name or leave blank to generate a name.`;
   } else {
     return undefined;
   }
```

Biome (2.5.1)

[error] This else clause can be omitted because previous branches break early.

(lint/style/noUselessElse)